### ISSUE TITLE
BACKTRACE 14973c window_cancel_textbox

### ISSUE THREAD

**Issue by IntelOrca** [OpenRCT2/OpenRCT2#8230]
<table>
    <tr>
        <td><img src='https://backtrace.io/images/icon.png' width=30 height=30 /></td>
        <td><a href='https://openrct2.sp.backtrace.io/dashboard/openrct2/project/openrct2/group/14973cd19ce5ee279b22f7a01cc24b05a9d054a10b64cc2e98d52e482d59e1ad?_t=W3sibmFtZSI6IndvcmtmbG93In1d'><b>New unique error: [openrct2]</b></a><br /><b>1</b> occurrence(s) <b></b> since (Tue Nov 06 2018 02:00:04)<br /></td>
    </tr>
</table>

<table>
<tr><td><strong>Application</strong></td><td>openrct2.exe</td></tr>

<tr><td><strong>Attributes</strong></td><td><strong>commit:</strong> df768a3: <em>1</em><br></td></tr>
<tr><td><strong>Classifiers</strong></td><td><strong>invalid-read</strong> </td></tr>
<tr><td><strong>Frames</strong></td><td><code>window_cancel_textbox -> input_widget_left -> game_handle_input_mouse -> game_handle_input -> context_handle_input -> OpenRCT2::GameState::Update -> OpenRCT2::Context::Update -> OpenRCT2::Context::RunGameLoop -> OpenRCT2::Context::RunOpenRCT2 -> NormalisedMain -> LaunchOpenRCT2</code></td></tr>
</table>

**Comment by janisozaur** [comment]
```
dump_path: /var/coronerd//openrct2/openrct2/_objects/0000/tx.0094.dmp outfile: -
Classifier: invalid-read
Crash info:
	Application: openrct2.exe
	Crashed YES
	reason: EXCEPTION_ACCESS_VIOLATION_READ
	address: 0
Memory Regions:
Sysinfo:
	arch: amd64 family 6 model 60 stepping 3
	cpu_count: 4
	os: Windows NT windows 10.0.17134 
Modules:
	0x7ff692e80000 - 0x7ff692eac000	openrct2.exe 
	0x7ffe0b830000 - 0x7ffe0c912000	openrct2.dll 
	0x7ffe13400000 - 0x7ffe13562000	FileSyncShell64.dll 18.192.920.12
	0x7ffe26620000 - 0x7ffe2663e000	DevDispItemProvider.dll 6.2.17134.1
	0x7ffe267b0000 - 0x7ffe268d7000	NetworkExplorer.dll 6.2.17134.1
	0x7ffe268e0000 - 0x7ffe26985000	tiptsf.dll 6.2.17134.191
	0x7ffe26b30000 - 0x7ffe26b55000	EhStorAPI.dll 6.2.17134.1
	0x7ffe26b60000 - 0x7ffe26bc3000	PlayToDevice.dll 6.2.17134.1
	0x7ffe26bd0000 - 0x7ffe26bdc000	davhlpr.dll 6.2.17134.1
	0x7ffe26be0000 - 0x7ffe26bfd000	davclnt.dll 6.2.17134.1
	0x7ffe26c00000 - 0x7ffe26c16000	ntlanman.dll 6.2.17134.1
	0x7ffe27ad0000 - 0x7ffe27b10000	IMJKAPI.DLL 6.2.17134.1
	0x7ffe27b10000 - 0x7ffe27b4d000	imkrapi.dll 6.2.17134.1
	0x7ffe27bf0000 - 0x7ffe27d9b000	dui70.dll 6.2.17134.112
	0x7ffe27da0000 - 0x7ffe27ef6000	IMETIP.DLL 6.2.17134.1
	0x7ffe27f00000 - 0x7ffe27fd1000	imkrtip.dll 6.2.17134.1
	0x7ffe290d0000 - 0x7ffe29198000	cscui.dll 6.2.17134.1
	0x7ffe291a0000 - 0x7ffe291d7000	EhStorShell.dll 6.2.17134.1
	0x7ffe29a30000 - 0x7ffe29a3b000	drprov.dll 6.2.17134.1
	0x7ffe29fe0000 - 0x7ffe29ff2000	cscapi.dll 6.2.17134.1
	0x7ffe2a060000 - 0x7ffe2a13a000	ntshrui.dll 6.2.17134.1
	0x7ffe2aa10000 - 0x7ffe2aa2d000	cldapi.dll 6.2.17134.1
	0x7ffe2abb0000 - 0x7ffe2ac0c000	thumbcache.dll 6.2.17134.319
	0x7ffe2acd0000 - 0x7ffe2ace4000	mskeyprotect.dll 6.2.17134.1
	0x7ffe2acf0000 - 0x7ffe2b185000	explorerframe.dll 6.2.17134.1
	0x7ffe2b1c0000 - 0x7ffe2b1e1000	ncryptsslp.dll 6.2.17134.137
	0x7ffe2b1f0000 - 0x7ffe2b248000	dataexchange.dll 6.2.17134.1
	0x7ffe2c960000 - 0x7ffe2c96c000	secur32.dll 6.2.17134.1
	0x7ffe2c990000 - 0x7ffe2c99b000	ktmw32.dll 6.2.17134.1
	0x7ffe2df00000 - 0x7ffe2df09000	IconCodecService.dll 6.2.17134.1
	0x7ffe2e920000 - 0x7ffe2ed8f000	wininet.dll 11.0.17134.345
	0x7ffe30e90000 - 0x7ffe30eb6000	srvcli.dll 6.2.17134.1
	0x7ffe32f70000 - 0x7ffe33007000	TextInputFramework.dll 6.2.17134.191
	0x7ffe33250000 - 0x7ffe3329a000	dlnashext.dll 6.2.17134.1
	0x7ffe333c0000 - 0x7ffe334ec000	AudioSes.dll 6.2.17134.137
	0x7ffe33b90000 - 0x7ffe33baa000	mpr.dll 6.2.17134.1
	0x7ffe353d0000 - 0x7ffe353f9000	dbgcore.dll 6.2.17134.1
	0x7ffe35400000 - 0x7ffe355c9000	dbghelp.dll 6.2.17134.1
	0x7ffe36e90000 - 0x7ffe36ed3000	icm32.dll 6.2.17134.1
	0x7ffe37230000 - 0x7ffe37240000	ColorAdapterClient.dll 6.2.17134.1
	0x7ffe37240000 - 0x7ffe374a9000	comctl32.dll 6.10.17134.345
	0x7ffe374b0000 - 0x7ffe37558000	mscms.dll 6.2.17134.1
	0x7ffe38440000 - 0x7ffe3844a000	rasadhlp.dll 6.2.17134.1
	0x7ffe39780000 - 0x7ffe397f2000	FWPUCLNT.DLL 6.2.17134.1
	0x7ffe3b760000 - 0x7ffe3b7cb000	oleacc.dll 7.2.17134.1
	0x7ffe3c120000 - 0x7ffe3c19f000	policymanager.dll 6.2.17134.191
	0x7ffe3c1a0000 - 0x7ffe3c216000	MMDevAPI.dll 6.2.17134.1
	0x7ffe3c6b0000 - 0x7ffe3c6ba000	version.dll 6.2.17134.1
	0x7ffe3c7d0000 - 0x7ffe3ce6b000	OneCoreUAPCommonProxyStub.dll 6.2.17134.1
	0x7ffe3d060000 - 0x7ffe3d0fc000	PortableDeviceApi.dll 6.2.17134.1
	0x7ffe3d170000 - 0x7ffe3d2bd000	WinTypes.dll 6.2.17134.112
	0x7ffe3d2c0000 - 0x7ffe3d5de000	CoreUIComponents.dll 6.2.17134.112
	0x7ffe3d7a0000 - 0x7ffe3d7aa000	avrt.dll 6.2.17134.1
	0x7ffe3d7b0000 - 0x7ffe3d95e000	WindowsCodecs.dll 6.2.17134.345
	0x7ffe3d960000 - 0x7ffe3d977000	wkscli.dll 6.2.17134.1
	0x7ffe3e0e0000 - 0x7ffe3e294000	propsys.dll 7.0.17134.112
	0x7ffe3e420000 - 0x7ffe3e433000	wtsapi32.dll 6.2.17134.1
	0x7ffe3e870000 - 0x7ffe3eb7b000	d3d11.dll 6.2.17134.112
	0x7ffe3f180000 - 0x7ffe3f31c000	dcomp.dll 6.2.17134.1
	0x7ffe3f320000 - 0x7ffe3f3b1000	msvcp110_win.dll 6.2.17134.1
	0x7ffe3f3c0000 - 0x7ffe3f49a000	CoreMessaging.dll 6.2.17134.345
	0x7ffe3f9c0000 - 0x7ffe3f9ea000	WINMMBASE.dll 6.2.17134.1
	0x7ffe3f9f0000 - 0x7ffe3fa13000	winmm.dll 6.2.17134.1
	0x7ffe3fa20000 - 0x7ffe3fa48000	mssprxy.dll 7.0.17134.285
	0x7ffe3fae0000 - 0x7ffe3fb6b000	apphelp.dll 6.2.17134.1
	0x7ffe3fc60000 - 0x7ffe3fcf8000	uxtheme.dll 6.2.17134.1
	0x7ffe3fee0000 - 0x7ffe3ff09000	dwmapi.dll 6.2.17134.1
	0x7ffe3fff0000 - 0x7ffe40011000	rmclient.dll 6.2.17134.81
	0x7ffe40070000 - 0x7ffe40228000	twinapi.appcore.dll 6.2.17134.137
	0x7ffe405e0000 - 0x7ffe4069b000	dxgi.dll 6.2.17134.112
	0x7ffe408f0000 - 0x7ffe40921000	ntmarta.dll 6.2.17134.1
	0x7ffe40960000 - 0x7ffe409b6000	winsta.dll 6.2.17134.1
	0x7ffe40bc0000 - 0x7ffe40c39000	schannel.dll 6.2.17134.319
	0x7ffe40c80000 - 0x7ffe40cb3000	rsaenh.dll 6.2.17134.254
	0x7ffe40e20000 - 0x7ffe40e58000	IPHLPAPI.DLL 6.2.17134.1
	0x7ffe40e60000 - 0x7ffe40f1e000	dnsapi.dll 6.2.17134.165
	0x7ffe40f20000 - 0x7ffe40f2e000	netutils.dll 6.2.17134.1
	0x7ffe410a0000 - 0x7ffe41106000	mswsock.dll 6.2.17134.1
	0x7ffe41250000 - 0x7ffe41267000	cryptsp.dll 6.2.17134.1
	0x7ffe41270000 - 0x7ffe4127b000	CRYPTBASE.dll 6.2.17134.1
	0x7ffe41310000 - 0x7ffe41346000	ntasn1.dll 6.2.17134.1
	0x7ffe41350000 - 0x7ffe41376000	ncrypt.dll 6.2.17134.1
	0x7ffe41380000 - 0x7ffe413a5000	bcrypt.dll 6.2.17134.112
	0x7ffe41630000 - 0x7ffe41657000	devobj.dll 6.2.17134.1
	0x7ffe41720000 - 0x7ffe41748000	userenv.dll 6.2.17134.1
	0x7ffe41750000 - 0x7ffe41780000	sspicli.dll 6.2.17134.1
	0x7ffe41820000 - 0x7ffe41831000	kernel.appcore.dll 6.2.17134.112
	0x7ffe41840000 - 0x7ffe4184a000	fltLib.dll 6.2.17134.1
	0x7ffe41850000 - 0x7ffe41862000	msasn1.dll 6.2.17134.1
	0x7ffe41870000 - 0x7ffe418bc000	powrprof.dll 6.2.17134.1
	0x7ffe418c0000 - 0x7ffe418df000	profapi.dll 6.2.17134.1
	0x7ffe418e0000 - 0x7ffe419da000	ucrtbase.dll 6.2.17134.319
	0x7ffe419e0000 - 0x7ffe41a7f000	msvcp_win.dll 6.2.17134.137
	0x7ffe41a80000 - 0x7ffe41cf3000	KERNELBASE.dll 6.2.17134.319
	0x7ffe41d00000 - 0x7ffe41d20000	win32u.dll 6.2.17134.1
	0x7ffe41dd0000 - 0x7ffe41f62000	gdi32full.dll 6.2.17134.345
	0x7ffe41f70000 - 0x7ffe41fb9000	cfgmgr32.dll 6.2.17134.1
	0x7ffe41fc0000 - 0x7ffe4203a000	bcryptPrimitives.dll 6.2.17134.1
	0x7ffe42040000 - 0x7ffe42097000	wintrust.dll 6.2.17134.81
	0x7ffe420a0000 - 0x7ffe42282000	crypt32.dll 6.2.17134.1
	0x7ffe42290000 - 0x7ffe4299d000	windows.storage.dll 6.2.17134.345
	0x7ffe429a0000 - 0x7ffe42af1000	ole32.dll 6.2.17134.137
	0x7ffe42b00000 - 0x7ffe42c74000	msctf.dll 6.2.17134.319
	0x7ffe42d20000 - 0x7ffe42dbe000	msvcrt.dll 7.0.17134.1
	0x7ffe42dc0000 - 0x7ffe42e60000	clbcatq.dll 2001.12.10941.16384
	0x7ffe42e60000 - 0x7ffe42f84000	rpcrt4.dll 6.2.17134.112
	0x7ffe42f90000 - 0x7ffe42fbd000	imm32.dll 6.2.17134.1
	0x7ffe42fc0000 - 0x7ffe42fe8000	gdi32.dll 6.2.17134.285
	0x7ffe42ff0000 - 0x7ffe43099000	SHCore.dll 6.2.17134.112
	0x7ffe43100000 - 0x7ffe43290000	user32.dll 6.2.17134.319
	0x7ffe43440000 - 0x7ffe4349b000	sechost.dll 6.2.17134.319
	0x7ffe434b0000 - 0x7ffe437d3000	combase.dll 6.2.17134.112
	0x7ffe437e0000 - 0x7ffe43831000	shlwapi.dll 6.2.17134.1
	0x7ffe43840000 - 0x7ffe438f2000	kernel32.dll 6.2.17134.1
	0x7ffe43960000 - 0x7ffe43a4d000	comdlg32.dll 6.2.17134.1
	0x7ffe43a50000 - 0x7ffe44e8f000	shell32.dll 6.2.17134.320
	0x7ffe44e90000 - 0x7ffe452db000	setupapi.dll 6.2.17134.1
	0x7ffe452e0000 - 0x7ffe4534c000	ws2_32.dll 6.2.17134.1
	0x7ffe45350000 - 0x7ffe453f1000	advapi32.dll 6.2.17134.319
	0x7ffe45400000 - 0x7ffe45408000	nsi.dll 6.2.17134.1
	0x7ffe45410000 - 0x7ffe454d2000	oleaut32.dll 6.2.17134.48
	0x7ffe45510000 - 0x7ffe456f1000	ntdll.dll 6.2.17134.254
Threads:
	thread-0 tid=3928 faulted
	[0] C:\Program Files\OpenRCT2\openrct2.dll!window_cancel_textbox() [c:\projects\openrct2-ject9\src\openrct2\interface\window.cpp : 2023 + 0]
	[1] C:\Program Files\OpenRCT2\openrct2.dll!input_widget_left(int,int,rct_window *,short) [c:\projects\openrct2-ject9\src\openrct2-ui\input\mouseinput.cpp : 1035 + 0x5]
	[2] C:\Program Files\OpenRCT2\openrct2.dll!game_handle_input_mouse(int,int,int) [c:\projects\openrct2-ject9\src\openrct2-ui\input\mouseinput.cpp : 303 + 0x11]
	[3] C:\Program Files\OpenRCT2\openrct2.dll!game_handle_input() [c:\projects\openrct2-ject9\src\openrct2-ui\input\mouseinput.cpp : 118 + 0xd]
	[4] C:\Program Files\OpenRCT2\openrct2.dll!context_handle_input() [c:\projects\openrct2-ject9\src\openrct2\context.cpp : 1243 + 0x9]
	[5] C:\Program Files\OpenRCT2\openrct2.dll!OpenRCT2::GameState::Update() [c:\projects\openrct2-ject9\src\openrct2\gamestate.cpp : 185 + 0x5]
	[6] C:\Program Files\OpenRCT2\openrct2.dll!OpenRCT2::Context::Update() [c:\projects\openrct2-ject9\src\openrct2\context.cpp : 936 + 0xc]
	[7] C:\Program Files\OpenRCT2\openrct2.dll!OpenRCT2::Context::RunGameLoop() [c:\projects\openrct2-ject9\src\openrct2\context.cpp : 801 + 0xd8]
	[8] C:\Program Files\OpenRCT2\openrct2.dll!OpenRCT2::Context::RunOpenRCT2(int,char const * *) [c:\projects\openrct2-ject9\src\openrct2\context.cpp : 213 + 0x8]
	[9] C:\Program Files\OpenRCT2\openrct2.dll!NormalisedMain(int,char const * *) [c:\projects\openrct2-ject9\src\openrct2-ui\ui.cpp : 60 + 0x13]
	[10] C:\Program Files\OpenRCT2\openrct2.dll!LaunchOpenRCT2(int,wchar_t * *) [c:\projects\openrct2-ject9\src\openrct2-dll\openrct2-dll.cpp : 46 + 0xf]
	[11] C:\Program Files\OpenRCT2\openrct2.exe!__scrt_common_main_seh() [f:\dd\vctools\crt\vcstartup\src\startup\exe_common.inl : 288 + 0x22]
	[12] C:\Windows\System32\kernel32.dll!BaseThreadInitThunk + 0x14
	[13] C:\Windows\System32\ntdll.dll!RtlUserThreadStart + 0x21
	thread-1 tid=8564 
	[0] C:\Windows\System32\ntdll.dll!NtWaitForWorkViaWorkerFactory + 0x14
		sp=0x917fdff678 bp=0 
	[1] C:\Windows\System32\ntdll.dll!TppWorkerThread + 0x536
	[2] C:\Windows\System32\kernel32.dll!BaseThreadInitThunk + 0x14
	[3] C:\Windows\System32\ntdll.dll!RtlUserThreadStart + 0x21
	thread-2 tid=7828 
	[0] C:\Windows\System32\ntdll.dll!NtGetContextThread + 0x14
		sp=0x917fefd7d8 
	[1] 0x1bf12a0eef0
	[2] C:\Windows\System32\ntdll.dll!RtlpAllocateHeapInternal + 0xeb
	[3] 0x1bf0000003b
	[4] C:\Windows\System32\ntdll.dll!LdrpResCompareResourceNames + 0xb9
	[5] C:\Windows\System32\ntdll.dll!LdrpResSearchResourceInsideDirectory + 0x1031
	[6] C:\Windows\System32\ucrtbase.dll!__crt_stdio_output::output_processor<wchar_t,__crt_stdio_output::string_output_adapter<wchar_t>,__crt_stdio_output::format_validation_base<wchar_t,__crt_stdio_output::string_output_adapter<wchar_t> > >::process + 0x236
	[7] C:\Windows\System32\ucrtbase.dll!common_vsprintf<__crt_stdio_output::format_validation_base,wchar_t> + 0x1a2
	thread-3 tid=9700 
	[0] C:\Windows\System32\ntdll.dll!NtWaitForWorkViaWorkerFactory + 0x14
		sp=0x917ffffaa8 bp=0 
	[1] C:\Windows\System32\ntdll.dll!TppWorkerThread + 0x536
	[2] C:\Windows\System32\kernel32.dll!BaseThreadInitThunk + 0x14
	[3] C:\Windows\System32\ntdll.dll!RtlUserThreadStart + 0x21
	thread-4 tid=9328 
	[0] C:\Windows\System32\ntdll.dll!NtWaitForWorkViaWorkerFactory + 0x14
		sp=0x91000ff948 bp=0 
	[1] C:\Windows\System32\ntdll.dll!TppWorkerThread + 0x536
	[2] C:\Windows\System32\kernel32.dll!BaseThreadInitThunk + 0x14
	[3] C:\Windows\System32\ntdll.dll!RtlUserThreadStart + 0x21
	thread-5 tid=656 
	[0] C:\Windows\System32\ntdll.dll!ZwWaitForMultipleObjects + 0x14
	[1] C:\Windows\System32\KERNELBASE.dll!WaitForMultipleObjectsEx + 0xf9
	[2] C:\Windows\System32\combase.dll!WaitCoalesced(void *,unsigned int,void * const *,unsigned long,unsigned long,int,unsigned long *) [onecore\com\published\comutils\coalescedwait.cxx : 70 + 0x25]
	[3] C:\Windows\System32\combase.dll!CROIDTable::WorkerThreadLoop(void *) [onecore\com\combase\dcomrem\refcache.cxx : 1649 + 0x30]
	[4] C:\Windows\System32\combase.dll!CRpcThread::WorkerLoop() [onecore\com\combase\dcomrem\threads.cxx : 269 + 0x1d]
	[5] C:\Windows\System32\combase.dll!CRpcThreadCache::RpcWorkerThreadEntry(void *) [onecore\com\combase\dcomrem\threads.cxx : 72 + 0x8]
	[6] C:\Windows\System32\kernel32.dll!BaseThreadInitThunk + 0x14
	[7] C:\Windows\System32\ntdll.dll!RtlUserThreadStart + 0x21
	thread-6 tid=7084 
	[0] C:\Windows\System32\ntdll.dll!NtWaitForWorkViaWorkerFactory + 0x14
		sp=0x91002ff618 bp=0 
	[1] C:\Windows\System32\ntdll.dll!TppWorkerThread + 0x536
	[2] C:\Windows\System32\kernel32.dll!BaseThreadInitThunk + 0x14
	[3] C:\Windows\System32\ntdll.dll!RtlUserThreadStart + 0x21
	thread-7 tid=4168 
	[0] C:\Windows\System32\ntdll.dll!NtWaitForWorkViaWorkerFactory + 0x14
		sp=0x91003ffbc8 bp=0 
	[1] C:\Windows\System32\ntdll.dll!TppWorkerThread + 0x536
	[2] C:\Windows\System32\kernel32.dll!BaseThreadInitThunk + 0x14
	[3] C:\Windows\System32\ntdll.dll!RtlUserThreadStart + 0x21
	thread-8 tid=4340 
	[0] C:\Windows\System32\ntdll.dll!NtWaitForWorkViaWorkerFactory + 0x14
		sp=0x91004ffae8 bp=0 
	[1] C:\Windows\System32\ntdll.dll!TppWorkerThread + 0x536
	[2] C:\Windows\System32\kernel32.dll!BaseThreadInitThunk + 0x14
	[3] C:\Windows\System32\ntdll.dll!RtlUserThreadStart + 0x21
	thread-9 tid=3212 
	[0] C:\Windows\System32\ntdll.dll!NtWaitForWorkViaWorkerFactory + 0x14
		sp=0x91005ffc18 bp=0 
	[1] C:\Windows\System32\ntdll.dll!TppWorkerThread + 0x536
	[2] C:\Windows\System32\kernel32.dll!BaseThreadInitThunk + 0x14
	[3] C:\Windows\System32\ntdll.dll!RtlUserThreadStart + 0x21
	thread-10 tid=6068 
	[0] C:\Windows\System32\ntdll.dll!NtWaitForWorkViaWorkerFactory + 0x14
		sp=0x91007ff4a8 bp=0 
	[1] C:\Windows\System32\ntdll.dll!TppWorkerThread + 0x536
	[2] C:\Windows\System32\kernel32.dll!BaseThreadInitThunk + 0x14
	[3] C:\Windows\System32\ntdll.dll!RtlUserThreadStart + 0x21
	thread-11 tid=5096 
	[0] C:\Windows\System32\ntdll.dll!NtWaitForWorkViaWorkerFactory + 0x14
		sp=0x91009ffaf8 bp=0 
	[1] C:\Windows\System32\ntdll.dll!TppWorkerThread + 0x536
	[2] C:\Windows\System32\kernel32.dll!BaseThreadInitThunk + 0x14
	[3] C:\Windows\System32\ntdll.dll!RtlUserThreadStart + 0x21
	thread-12 tid=7708 
	[0] C:\Windows\System32\ntdll.dll!NtWaitForWorkViaWorkerFactory + 0x14
		sp=0x9100affc58 bp=0 
	[1] C:\Windows\System32\ntdll.dll!TppWorkerThread + 0x536
	[2] C:\Windows\System32\kernel32.dll!BaseThreadInitThunk + 0x14
	[3] C:\Windows\System32\ntdll.dll!RtlUserThreadStart + 0x21
	thread-13 tid=2484 
	[0] C:\Windows\System32\ntdll.dll!ZwWaitForAlertByThreadId + 0x14
	[1] C:\Windows\System32\ntdll.dll!RtlSleepConditionVariableSRW + 0x109
	[2] C:\Windows\System32\KERNELBASE.dll!SleepConditionVariableSRW + 0x32
	[3] C:\Program Files\OpenRCT2\openrct2.dll!__crtSleepConditionVariableSRW(_RTL_CONDITION_VARIABLE *,_RTL_SRWLOCK *,unsigned long,unsigned long) [f:\dd\vctools\crt\crtw32\misc\winapisupp.cpp : 657 + 0x6]
	[4] C:\Program Files\OpenRCT2\openrct2.dll!Concurrency::details::stl_condition_variable_win7::wait_for(Concurrency::details::stl_critical_section_interface *,unsigned int) [f:\dd\vctools\crt\crtw32\stdcpp\thr\primitives.h : 216 + 0x10]
	[5] C:\Program Files\OpenRCT2\openrct2.dll!do_wait(_Cnd_internal_imp_t *,_Mtx_internal_imp_t *,xtime const *) [f:\dd\vctools\crt\crtw32\stdcpp\thr\cond.c : 82 + 0x26]
	[6] C:\Program Files\OpenRCT2\openrct2.dll!std::_LaunchPad<std::unique_ptr<std::tuple<<lambda_f574185c45dda01958beedcd6f2ea127> >,std::default_delete<std::tuple<<lambda_f574185c45dda01958beedcd6f2ea127> > > > >::_Go() [c:\program files (x86)\microsoft visual studio\2017\community\vc\tools\msvc\14.14.26428\include\thr\xthread : 230 + 0xf8]
	[7] C:\Program Files\OpenRCT2\openrct2.dll!std::_Pad::_Call_func(void *) [c:\program files (x86)\microsoft visual studio\2017\community\vc\tools\msvc\14.15.26726\include\thr\xthread : 208 + 0x5]
	[8] C:\Program Files\OpenRCT2\openrct2.dll!thread_start<unsigned int (*)(void *)>(void * const) [d:\rs1\minkernel\crts\ucrt\src\appcrt\startup\thread.cpp : 115 + 0xd]
	[9] C:\Windows\System32\kernel32.dll!BaseThreadInitThunk + 0x14
	[10] C:\Windows\System32\ntdll.dll!RtlUserThreadStart + 0x21
	thread-14 tid=3448 
	[0] C:\Windows\System32\ntdll.dll!ZwWaitForSingleObject + 0x14
	[1] C:\Windows\System32\KERNELBASE.dll!WaitForSingleObjectEx + 0xa2
	[2] C:\Program Files\OpenRCT2\openrct2.dll!WASAPI_WaitDevice(SDL_AudioDevice *) [c:\projects\dependencies\vcpkg\buildtrees\sdl2\src\ease-2.0.8-42957b8d01\src\audio\wasapi\sdl_wasapi.c : 331 + 0xe]
	[3] C:\Program Files\OpenRCT2\openrct2.dll!SDL_RunAudio(void *) [c:\projects\dependencies\vcpkg\buildtrees\sdl2\src\ease-2.0.8-42957b8d01\src\audio\sdl_audio.c : 711 + 0x9]
	[4] C:\Program Files\OpenRCT2\openrct2.dll!SDL_RunThread(void *) [c:\projects\dependencies\vcpkg\buildtrees\sdl2\src\ease-2.0.8-42957b8d01\src\thread\sdl_thread.c : 283 + 0x5]
	[5] C:\Program Files\OpenRCT2\openrct2.dll!RunThreadViaBeginThreadEx(void *) [c:\projects\dependencies\vcpkg\buildtrees\sdl2\src\ease-2.0.8-42957b8d01\src\thread\windows\sdl_systhread.c : 105 + 0x14]
	[6] C:\Program Files\OpenRCT2\openrct2.dll!thread_start<unsigned int (*)(void *)>(void * const) [d:\rs1\minkernel\crts\ucrt\src\appcrt\startup\thread.cpp : 115 + 0xd]
	[7] C:\Windows\System32\kernel32.dll!BaseThreadInitThunk + 0x14
	[8] C:\Windows\System32\ntdll.dll!RtlUserThreadStart + 0x21
	thread-15 tid=2140 
	[0] C:\Windows\System32\ntdll.dll!NtWaitForWorkViaWorkerFactory + 0x14
		sp=0x9100dffc38 bp=0 
	[1] C:\Windows\System32\ntdll.dll!TppWorkerThread + 0x536
	[2] C:\Windows\System32\kernel32.dll!BaseThreadInitThunk + 0x14
	[3] C:\Windows\System32\ntdll.dll!RtlUserThreadStart + 0x21
	thread-16 tid=4612 
	[0] C:\Windows\System32\ntdll.dll!NtWaitForWorkViaWorkerFactory + 0x14
		sp=0x9100eff9f8 bp=0 
	[1] C:\Windows\System32\ntdll.dll!TppWorkerThread + 0x536
	[2] C:\Windows\System32\kernel32.dll!BaseThreadInitThunk + 0x14
	[3] C:\Windows\System32\ntdll.dll!RtlUserThreadStart + 0x21
	thread-17 tid=1032 
	[0] C:\Windows\System32\ntdll.dll!NtRemoveIoCompletion + 0x14
	[1] C:\Windows\System32\mswsock.dll!SockAsyncThread + 0x88
	[2] C:\Windows\System32\kernel32.dll!BaseThreadInitThunk + 0x14
	[3] C:\Windows\System32\ntdll.dll!RtlUserThreadStart + 0x21
	thread-18 tid=1036 
	[0] C:\Windows\System32\ntdll.dll!NtWaitForWorkViaWorkerFactory + 0x14
		sp=0x9100fff9a8 bp=0 
	[1] C:\Windows\System32\ntdll.dll!TppWorkerThread + 0x536
	[2] C:\Windows\System32\kernel32.dll!BaseThreadInitThunk + 0x14
	[3] C:\Windows\System32\ntdll.dll!RtlUserThreadStart + 0x21
```

**Comment by ZehMatt** [comment]
Probably fixed by one of #9470, #9738, #9330
